package.json

```diff
@@ -209,9 +210,9 @@
     "eslint-fix": "yarn && eslint --fix --ext .js --ext .jsx --rulesdir frontend/lint/eslint-rules frontend/src frontend/test",
     "docs": "documentation serve --watch frontend/src/metabase-lib/lib/**",
     "ci": "yarn ci-frontend && yarn ci-backend",
-    "ci-frontend": "yarn lint && yarn flow && yarn test",
+    "ci-frontend": "yarn lint && yarn test",
     "ci-backend": "lein docstring-checker && lein bikeshed && lein eastwood && lein test",
-    "test-cypress-headless": "yarn build && ./bin/build-for-test && yarn test-cypress-no-build",
+    "test-cypress": "yarn build && ./bin/build-for-test && yarn test-cypress-no-build",
```

Comment:
@paulrosenzweig I think we can consider "headless" to be implied / the default. Is that ok with you?

Reply:
👍 sounds good.